superset/databases/dao.py

```diff
@@ -124,3 +125,13 @@ def get_related_objects(cls, database_id: int) -> Dict[str, Any]:
         return dict(
             charts=charts, dashboards=dashboards, sqllab_tab_states=sqllab_tab_states
         )
+
+    @classmethod
+    def get_related_ssh_tunnel(cls, database_id: int) -> Dict[str, Any]:
```

Comment:
let's just call the `get_ssh_tunnel` since it's a 1 to 1 relationship

Reply:
Cool! Renamed to `get_ssh_tunnel` 👍